workflow-examples/src/main/java/org/parodos/workflow/examples/task/SimpleWorkFlowCheckerTask.java

```diff
@@ -0,0 +1,34 @@
+package org.parodos.workflow.examples.task;
+
+import com.redhat.parodos.workflow.task.checker.BaseWorkFlowCheckerTask;
+import com.redhat.parodos.workflow.task.enums.WorkFlowTaskOutput;
+import com.redhat.parodos.workflow.task.parameter.WorkFlowTaskParameter;
+import com.redhat.parodos.workflows.work.DefaultWorkReport;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkReport;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j
+public class SimpleWorkFlowCheckerTask extends BaseWorkFlowCheckerTask {
+
+	@Override
+	public WorkReport checkWorkFlowStatus(WorkContext workContext) {
+		log.info("NamespaceApprovalWorkFlowCheckerTask");
+		return new DefaultWorkReport(WorkStatus.FAILED, workContext);
+	}
+
+	@Override
+	public List<WorkFlowTaskParameter> getWorkFlowTaskParameters() {
+		return Collections.emptyList();
+	}
+
+	@Override
+	public List<WorkFlowTaskOutput> getWorkFlowTaskOutputs() {
+		return Collections.emptyList();
+	}
```

Comment:
nit: no need to override these methods because they are already implemented here https://github.com/redhat-developer/parodos/blob/de2451a3e499bf2ea31e5eeae221e3c5782c00b4/parodos-model-api/src/main/java/com/redhat/parodos/workflow/task/WorkFlowTask.java#L44-L57